### ISSUE TITLE
Get new mirrors and lenses for Gerald

### ISSUE THREAD

**Issue by JackiePease** [DoESLiverpool/somebody-should#1443]
Get some new mirrors and lenses for Gerald to replace the current ones which have been worn out by the Covid project

**Comment by Sean-anotherone** [comment]
May I propose that this might be an opportune moment to think about some upgrades?
Such as :

https://www.ebay.co.uk/itm/E-Series-1st-Mirror-Mount-Beam-Combiner-Included-for-CO2-Laser-Head/123589673559

To provide a targeting spot?

Sean-anotherone

**Comment by Sean-anotherone** [comment]
I did a good clean and it was working again so I suggest getting a set once the covid project is over - if they survive that long !

**Comment by magman2112** [comment]
I gave Gerald a good clean today and noticed a couple of cracks in the lens. Fortunately, these are outside the centre of the lens, so OK for now, if the laser is aligned correctly. It will cause more issues as the alignment drifts.

I would suggest getting a spare lens as well and try to keep a spare in the future, to reduce downtime when they fail.

We also need replacement filters for the exhaust, as both of the existing ones are broken.

**Comment by Sean-anotherone** [comment]
Here is my list of the things Gerald needs - feel free to edit/add as anyone sees fit.

1. Mirrors - I thing the covid masks have killed them, 20mm, 2 or 3 required depending on if item 2 is purchased.
2. The new mount for the primary mirror - this has become really quite difficult to adjust - and it you do adjust it well it moves when the bolts are tightened.  Replacement required.  My post from 4 may refers.
3. final focus lens - 50.8mm FL, I forgot to measure the diameter but it's about less thasn the 20mm mirrors.  this is both damaged (cracks reported by Mike) and messed up with polypropylene.  Planoconvex is best, curved side goes up.
4. New bearings and slides for X and Y axes.  Since the covid project the X sled bearings are now giving about 2mm of slack, should be basically unmeasurable.  Mike has previously bought Y slides and bearings but they need work before they can be fitted.
5. new belt for both y Axes drives and X axis
6. z axis jacks need strip clean and reassembly as a bare minimum, new belts as well, but a complete modification with better drive geometry is the way to go.  The jacks are probably reusable once the pp is cleaned off, but the belt is just too long and convoluted to use a single motor  - it skips teeth regularly and that leaves the bed out of level.
7. Once monty is finished (still need to decide the way forward for the laser) another cohesion controller would be good for Gerald and give access to lightburn ( a third cohesion would drop right into Sophia too for a common, much more modern platform.
8. meed more network ports in laser room, or at least longer cables.
9. the tube is certainly down on power, not dead yet, but many hours of cutting pp have taken a heavy toll.
10.  The air assist pump needs a service, it hardly blows.
11. The air assist needs new tube all the way from the inlet to the nozzle.


In short, Gerald was already a bit ropey and the visor project has left it hanging by a thread.  without at least some of the above it's just a matter of time until the next big failure.

Sorry if that's a bit of a rant but enthusiasm and home made bits will only take it so far.....

**Comment by magman2112** [comment]
I agree with all of Sean’s comments but I would add the following.

12. Replace some of the key wiring and fix it properly to the chassis. Some cables, like the lid interlock, have cracks in the insulation and are hanging loose in the machine, these cables will fail in the future.
13. We need limit switches on both ends of the x and y axis, as common alignment issues crop up after banging into the ends of the axes, possibly due to inexperienced users - which we are always likely to have. The machine should be safer and easier to use for all users.
14. Lower priority, but it would be useful to have the home button working, to give a reference to start cutting from.
15. Change the fluorescent strip to LED lighting and significantly improve the illumination bed, as this will help with user training and general use. 
16. Let’s investigate And record the minimum power levels that are needed for the the laser to fire, as we currently have to guess this, when we should know. This is especially relevant to the corner power settings. I suspect anything less than 10% power, doesn’t really switch on the laser properly.
17. Lets get some real performance specifications for the laser and the laser power supply, to see if they are fully compatible and in case we are not running the optimum settings.
18. Fit more draft excluders in panel gaps to improve airflow across the bed, to consequently improve smoke removal.

If I can think of anything else, I will add it later.

**Comment by Sean-anotherone** [comment]
Gerald still wants one of these :  https://www.ebay.co.uk/itm/E-Series-1st-Mirror-Mount-Beam-Combiner-Included-for-CO2-Laser-Head/123589673559

**Comment by amcewen** [comment]
One of those ordered.  Estimated delivery 11-18 Aug apparently.

**Comment by amcewen** [comment]
I've also ordered a new [II-VI Infrared ZnSe 63.5mm Focal Length Lens 20mm OD](https://smokeandmirrors.store/products/iivi-20mm-znse-63-3mm-focal-length-lens) for Gerald and a spare [II-VI Infrared ZnSe 50.8mm Focal Length Lens 20mm OD](https://smokeandmirrors.store/products/copy-of-ii-vi-infrared-znse-63-5mm-focal-length-lens-20mm-od?variant=48209787210) for Sophia, and a new set of [20mm mirrors](https://smokeandmirrors.store/products/economy-20mm-od-mo-mirror?variant=8525426524196) for Gerald (who needs 3) plus a spare.  (I was going to get a set for Sophia too, but Smoke and Mirrors seems out-of-stock.  I've pinged Dominic to double-check)

**Comment by amcewen** [comment]
Additional 3 mirrors ordered too, so we'll have a full replacement set for Gerald and Sophia.

**Comment by amcewen** [comment]
Mirrors and lenses have arrived.  They're on @DoESsean's desk for safe-keeping.

And I've paid UPS the £20.17 import fees for the beam combiner.

**Comment by amcewen** [comment]
The beam combiner has arrived.  It's in its box next to the lenses on @DoESsean's desk.

**Comment by magman2112** [comment]
@Sean-anotherone and I fitted the new beam combiner and primary mirror yesterday evening. This turned into a bit more of a task than we originally anticipated, but we got it working quite well, though we may need to adjust it a little more when we have some more time.

Gerald was going out of alignment prior to fitting the beam combiner anyway, but we have now completed a full alignment. Again, we can possibly improve this alignment with a bit more tuning, but people were queuing up to use Gerald and the alignment was better than when we started, so we will see how it behaves over the next couple of days.

Having the new visible laser is already getting positive feedback, so a job well done methinks.

I will close this issue for now, though I will likely pick up some of the issues raised above and append them to the a Gerald upgrade issue which I believe is already open.
